.claude/scripts/claude-md-compliance.py

```
@@ -0,0 +1,722 @@
+            modified.append(_posix(path))
+    return modified
+
+
+# ---------------------------------------------------------------------------
+# File collection
+# ---------------------------------------------------------------------------
+
+
+def get_staged_files() -> list[Path]:
+    result = subprocess.run(
+        ["git", "diff", "--cached", "--name-only", "--diff-filter=ACM", "--", "*.py"],
```

Ruff (0.15.1)

[error] Starting a process with a partial executable path

(S607)